ccp/vars

```diff
@@ -1,19 +1,16 @@
-BROKER_ID=broker.dev.ccp-it.dktk.dkfz.de
+BROKER_ID=broker.ccp-it.dktk.dkfz.de
```

Comment:
Please don't merge this to main, we're still in the process of getting all sites to the new broker. Until all sites are migrated, we need to move the already migrated to dktk-migration branch.